### ISSUE TITLE
1.4 -> 1.5 Update doesn't seem to install

### ISSUE THREAD

**Issue by swebb99uk** [Revivius/nb-darcula#98]
When I run up Netbeans (8.1) it tells me that there is an update available. I go through the normal steps to update and restart the IDE and it still tells me there is an update available. I don't see any kind of errors.

**Comment by Revivius** [comment]
@swebb99uk; update works ok for me. Problem persists ?

**Comment by swebb99uk** [comment]
Hi there,

I tried a few times and it didn't update. However I just tried again as there was a list of other updates and they all failed initially. BUT I have been running netbeans via a shortcut icon on my ubuntu 15.10 box and I thought to myself what happens if I run up netbeans via './netbeans' rather than the shortcut, it has now updated everything after showing me the update wizard as it started!!!!

So it is working now :) Not sure what the problem was with starting from a shortcut.
